components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -502,6 +504,7 @@ - (void)setTitleBelowItem:(BOOL)titleBelowItem {
 
 - (void)setSelectedItemTintColor:(UIColor *)selectedItemTintColor {
   _selectedItemTintColor = selectedItemTintColor;
+  _selectedItemTitleColor = selectedItemTintColor;
```

Comment:
Should this call setSelectedItemTitleColor?

Reply:
Nope, this should take care of everything. Setting itemView.selectedItemTintColor updates the itemView's text label color